product_docs/docs/pem/8/pem_online_help/pem_worker_usage_guide.mdx

```diff
@@ -9,155 +9,154 @@ The PEM Worker performs tasks on behalf of the PEM Agent. Please note that all t
+    --webhook-ssl-ca-crt (The complete path to the webhook SSL CA certificate file.)
+    --webhook-ssl-crl (The complete path of the CRL file to validate revoked webhook server certificate.)
+    --webhook-allow-insecure-ssl (Default: false, Allow webhooks to call with insecure flag.)
+    --config-dir <Directory path of agent configuration file (Default: "<pemworker path>/../etc")
+
+  remove-webhook-options:
+    --webhook-name (Name of the webhook endpoint.)
+    --config-dir <Directory path of agent configuration file (Default: "<pemworker path>/../etc")
+
+  barman-registration-options:
+    --api-url <url> (URL of the BARMAN API 'pg-backup-api')
+    --description <name> (Description to show on the UI 'User interface' for the BARMAN API.)
+    --probe-execution-frequency <interval> (Default: 30, Probe the BARMAN API 'pg-backup-api' at regular interval 'in seconds' and fetch the metrics.)
+    --heartbeat-interval <interval> (Default: 10, Ping the BARMAN API 'pg-backup-api' 'status' API at a regular interval 'in seconds' for checking its availability.)
+    --ssl-crt <certificate_file> (SSL certificate file for the BARMAN API.)
+    --ssl-key <key_file> (Private SSL key for the BARMAN API.)
+    --ssl-ca-cert <ca_file> (CA certificate to verify peer against the BARMAN API.)
+    --team <database-role> (Specify the name of the database group role, on the PEM backend database server, that should have access to this BARMAN API Server.)
+    --owner <database-user> (Specify the name of the database user, on the PEM backend database server, who will own the BARMAN API Server.)
+    --config-file/-c <config_file> (Path to the agent configuration file.)
+
+  barman-update-options:
+    --id <barman-id> (ID for the existing BARMAN API 'pg-backup-api')
+    --api-url <url> (URL of the BARMAN API 'pg-backup-api')
+    --probe-execution-frequency <interval> (Default: 30, Probe the BARMAN API 'pg-backup-api' at regular interval 'in seconds' and fetch the metrics.)
+    --heartbeat-interval <interval> (Default: 10, Ping the BARMAN API 'pg-backup-api' 'status' API at a regular interval 'in seconds' for checking its availability.)
+    --ssl-crt <certificate_file> (SSL certificate file for the BARMAN API.)
+    --ssl-key <key_file> (Private SSL key for the BARMAN API.)
+    --ssl-ca-cert <ca_file> (CA certificate to verify peer against the BARMAN API.)
+    --config-file/-c <config_file> (Path to the agent configuration file.)
+
+  barman-unregistration-options:
+    --id <barman-id> (ID for the existing BARMAN API, registered with the PEM Server.'pg-backup-api')
+    --config-file/-c <config_file> (Path to the agent configuration file.)
+
+    NOTE:
+      Set environment variable PEM_SERVER_PASSWORD to provide the password for the
+      PEM server to allow the pemworker to connect as a PEM admin user.
+```
+
+##  On Windows:  
+
+    ```shell-session
```

Comment:
Drop the indentation here now that these are in sections and not a list

Reply:
Reworked into H3/H4 sections for better readability.